cfgorch/cfgorch.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <map>
+#include <memory>
+#include "dbconnector.h"
+#include "table.h"
+#include "consumertable.h"
+
+using namespace std;
+using namespace swss;
+
+#define DEFAULT_KEY_SEPARATOR           ":"
+#define CONFIGDB_KEY_SEPARATOR          "|"
+
+typedef map<string, KeyOpFieldsValuesTuple> SyncMap;
+struct Consumer {
+    Consumer(TableConsumable* consumer) : m_consumer(consumer)  { }
+    TableConsumable* m_consumer;
+    /* Store the latest 'golden' status */
+    SyncMap m_toSync;
+};
+typedef pair<string, Consumer> ConsumerMapPair;
+typedef map<string, Consumer> ConsumerMap;
+
+class CfgOrch
```

Comment:
>CfgOrch [](start = 6, length = 7)

inherit from Orch?

Reply:
Orch looks too heavy for config orchestration, but cfgorch does have major overlap with orch.  Will look into it further to see how to reduce the duplication.